scripts/joint_trajectory_file_playback.py

```diff
@@ -110,6 +110,8 @@ def __init__(self):
         self._l_grip = FollowJointTrajectoryGoal()
         self._r_grip = FollowJointTrajectoryGoal()
 
+        # Timing offset to prevent gripper playback before trajectory has started
+        self._trajectory_start_offset = rospy.Duration(0.0)
```

Comment:
Yes, the value here is overwritten as soon as the proper `start_offset` is calculated.